packages/leafygreen-provider/src/LeafyGreenProvider.tsx

```diff
@@ -1,13 +1,24 @@
 import React from 'react';
 import PropTypes from 'prop-types';
 import UsingKeyboardProvider from './UsingKeyboardProvider';
+import BaseFontSizeProvider from './BaseFontSizeProvider';
 
 interface LeafyGreenProviderProps {
   children: React.ReactNode;
+  baseFontSize?: number;
```

Comment:
We should type this as an enum – we only want to handle `14 | 16` right now.

Reply:
👍